aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,285 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization: Localization and Globalization 
```

Comment:
Remove "ASP.NET Core" from the title. How about just "Globalization and Localization", which mirrors the [MSDN](https://msdn.microsoft.com/en-us/library/aa292205(VS.71).aspx) treatment of this topic.